### ISSUE TITLE
Editorial: Revocable sequence maturity contract: clarify term

### ISSUE THREAD

**Issue by s-tikhomirov** [lnbook/lnbook#460]
**Which part needs editing?**
The term "revocable sequence maturity contract" is hard to grasp (for me, at least). From what I understood, it is the transaction used to open a payment channel. However, its name looks as if it was made complex just for the sake of sounding smart.

**What editing does this need?**
Why is it called RSMC? I kinda get what "revocable" refers to, but what "sequence" are we talking about? When (and how) does it "mature"?

**Comment by Roasbeef** [comment]
Yeah it's an out of date term from the original white paper. That existing construct actually isn't also used at all in the current version of the spec, nor in any of the LN codebases. IMO it should be removed, and we can just describe the contracts as having a delay in order to allow proper arbitration within the time frame, etc.

**Comment by renepickhardt** [comment]
has that happened?
